src/pages/OurTeamPage.tsx

```diff
@@ -1,9 +1,32 @@
+import { Avatar, Divider, Space, Typography } from 'antd';
 import React from 'react';
+import { content, MemberCardProps } from '../content/OurTeamContent';
+import styles from '../styles/OurTeamPage.module.less';
+
+const MemberCard: React.FC<MemberCardProps> = (props) => {
+  return (
+    <div className={styles.ourteampage_member}>
+      <Avatar className={styles.ourteampage_avatar} size={150} src={props.imgSrc} />
+      <div className={styles.ourteampage_text}>
+        <Typography.Title level={3}>{props.title}</Typography.Title>
+        <Typography.Text>{props.subtitle}</Typography.Text>
+      </div>
+    </div>
+  );
+};
 
 const OurTeamPage: React.FC = () => {
   return (
-    <div>
-      <p>Team Page</p>
+    <div className={styles.ourteampage}>
+      <Divider className={styles.teampagedivider} />
+      <div className={styles.myteamtitle}>
+        <Typography.Title>OUR TEAM</Typography.Title>
```

Comment:
Maybe try a slightly different level. Looks like level 3 is about 44px and we want something closer to 35px (2.5rem). Really, we just need to standardize header sizes etc with @minkyoseo